templates/base.html

```diff
@@ -63,6 +63,7 @@
     <meta property="og:title" content="{{ title|safe }}">
     <meta property="og:url" content="{{app.site.url}}{{ request.path }}">
     <meta property="og:language" content="fr_FR">
+    <meta property="og:image" content="https://{{ meta_image }}">
```

Comment:
J'ai trouvé dans la documentation "opengraph": 

URL | A sequence of Unicode characters that identify an Internet resource. | All valid URLs that utilize the http:// or https:// protocols

Donc ça me parait une bonne idée de le préciser.